src/OddJobs/templates/OddJobs/customeractive.html

```diff
@@ -12,40 +12,15 @@
     <title>Odd Jobs</title>
 </head>
 <body>
-    <!-- Top Navigation -->
     {% include 'OddJobs/top_nav.html' %}
-
     <div class="container my-5">
-        <div class="row">
-            <div class="col-lg-4 mt-3 mt-lg-0">
-                <div class="card">
-                    <div class="card-body">
-                        <h2 class="h4 card-title">Cool Cat 1</h2>
-                        <p class="card-text">Some quick example text to build on the card title and make up the bulk of the card's content.</p>
-                        <div class="text-end"><a href="#" class="btn btn-primary">Go somewhere</a></div>
-                    </div>
-                </div>
-            </div>
-            <div class="col-lg-4 mt-3 mt-lg-0">
-                <div class="card">
-                    <div class="card-body">
-                        <h2 class="h4 card-title">Cool Cat 2</h2>
-                        <p class="card-text">Some quick example text to build on the card title and make up the bulk of the card's content.</p>
-                        <div class="text-end"><a href="#" class="btn btn-primary">Go somewhere</a></div>
-                    </div>
-                </div>
-            </div>
-            <div class="col-lg-4 mt-3 mt-lg-0">
-                <div class="card">
-                    <div class="card-body">
-                        <h2 class="h4 card-title">Cool Cat 3</h2>
-                        <p class="card-text">Some quick example text to build on the card title and make up the bulk of the card's content.</p>
-                        <div class="text-end"><a href="#" class="btn btn-primary">Go somewhere</a></div>
-                    </div>
-                </div>
-            </div>
+        <div class="row" id="cardHolder">
         </div>
     </div>
+    <script type="text/javascript">
+      var jobData = {{ jobs|safe }}
+    </script>
+    <script type="text/javascript" src="{% static 'OddJobs/jobcardsactive.js' %}"></script>
     <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.bundle.min.js" integrity="sha384-ka7Sk0Gln4gmtz2MlQnikT1wXgYsOg+OMhuP+IlRH9sENBO0LRn5q+8nbTov4+1p" crossorigin="anonymous"></script>
```

Comment:
This line should not be here. The dropdowns in the top navigation won't work if it is. (this script is included with the top navigation)

Reply:
Line 23